modules/shared/components/molecules/Navigation/Navigation.tsx

```diff
@@ -16,90 +19,107 @@ import { useRedirect } from '../../../hooks/useRedirect/useRedirect';
 import { useAuth } from '../../../../../context/AuthUserContext/AuthUserContext';
 import DropDown from '../../atoms/DropDown/DropDown';
 
+const homeNavLinks = [
+  {
+    name: 'home',
+    path: '/',
+    content: <HomeIcon />,
+    active: true,
+  },
+  {
+    name: 'friends',
+    path: '/',
+    content: <FriendsIcon />,
+  },
+  {
+    name: 'bill',
+    path: '/',
+    content: <BillIcon />,
+  },
+];
+
+const userNavLinks = [
+  {
+    name: 'help',
+    path: '/',
+    content: <HelpIcon />,
+  },
+  {
+    name: 'happy',
+    path: '/',
+    content: <HappyIcon />,
+  },
+];
+
 const Navigation: FC = (): ReactElement => {
+  const { pathname } = useRouter();
   const { user } = useAuth();
   const { redirectToLoginPage } = useRedirect();
-
   const logout = async (): Promise<void> => {
     try {
       await logoutUser();
       redirectToLoginPage();
     } catch (err: unknown) {
-      console.log('Logo out access is denied', err);
+      toast.error('Logo out access is denied');
     }
   };
   const avatarVariant = user?.userImg ? 'filled' : 'notFilled';
 
   return (
     <nav className={styles['navigation-wrapper']}>
       <div className={styles.navigation}>
-        <div className="brand-name-container">
+        <div>
           <Link href="/">
-            <a className={styles['brand-name']}>Pickify</a>
+            <a>
+              <Logo />
+            </a>
           </Link>
         </div>
-        <div className={styles['nav-links-container']}>
-          <div className={styles['home-nav-links']}>
-            <ul>
-              <li>
-                <Link href="/">
-                  <a>
-                    <HomeIcon />
-                  </a>
+        <div className={styles['nav-links']}>
+          <ul>
+            {homeNavLinks.map((homeNavItem) => (
```

Comment:
great work, by adding all the links in an array and make the JSX dynamic like that, it will be easier to add any link in the future and also made the code more cleaner, nice approach.